fiatconnect-api.md

```diff
@@ -385,6 +386,48 @@ will be allowed to initiate a transfer. In the case where a user submits KYC det
 provider, they will never be able to initiate a transfer, regardless of what location they submit to the quote endpoint, since the KYC
 verification should be denied.
 
+## 3.2. Clock Synchronization
+
+Various parts of the FiatConnect specification rely on the client and server synchronizing their clocks in order to agree on a common time.
+In particular, the authentication scheme relies on clock synchronization to choose proper issuance and expiration times for session cookies,
+and the quote endpoints return timestamps representing the expiry of the requested quote, which requires client-server clock synchronization
+in order to be interpreted correctly.
+
+Rather than having bespoke clock synchronization solutions for each of these cases, the FiatConnect specification requires API implementors to
+include a single `/clock` endpoint, which simply returns the server's current timestamp. Servers MUST implement this endpoint. With this information,
+the client can determine the offset between their local time and server time, and synchronize on clocks (to an acceptable approximation).
+The exact method by which clients determine the clock offset is an implementation detail left to the client. (For example, a relatively
+easy and inexpensive method that incoroporates round-trip delay is the approach taken by the
+[Network Time Protocol](https://en.wikipedia.org/wiki/Network_Time_Protocol#Clock_synchronization_algorithm).)
```

Comment:
to actually use this algo you'd need more information-- in particular, there's no way to get t1 (the time the request hits the server) from the `/clock` endpoint as currently written.

Reply:
although I guess you could assume t1 and t2 are both equal to the `time` value on the response body

Reply:
Yeah, I also thought about this and came to the same assumption, but didn't want to be too prescriptive/go into too much detail here. Realistically t1 and t2 will be extremely close to each other anyways. Presumably some clients won't even incorporate RTT into their offset, so just providing a pointer to this here seems like enough for curious client developers.